### PR TITLE
Make easy for custom validation style.

### PR DESCRIPTION
**Pull request by nguyenntuong**
Remove unuse method.

Summary of the changes (Less than 80 chars)
 - Change render build order to make custom css is possible.
 - Remove unsuse method HandleValidationStateChanged.

### CODE REVIEW

**Comment by pranavkm** [comment]
@nguyenntuong thanks for the PR. Could you add some tests for your new property?

File: src/Components/Web/src/Forms/ValidationMessage.cs [hunk 1]
```diff
@@ -68,23 +73,19 @@ protected override void OnParametersSet()
         }
 
         /// <inheritdoc />
+        /// make easy for custom validation style
```
Comment (pranavkm) [inline src/Components/Web/src/Forms/ValidationMessage.cs]: Please remove

**Comment by nguyenntuong** [comment]
> @nguyenntuong thanks for the PR. Could you add some tests for your new property?
Can you give me some tests template ? Actually, i am not professional.

**Comment by pranavkm** [comment]
Thanks for your effort @nguyenntuong. Unfortunately we are unable to accept this PR without some tests and we don't have the resources to help you with at this time.

**Pull request closed without merging**